api/api.md

```diff
@@ -1,6 +1,9 @@
 # REST API
 
-This section documents part of the REST API provided by the Gravwell webserver. For complete, up-to-date information about the REST API, please refer to the Gravwell Go Client <https://pkg.go.dev/github.com/gravwell/gravwell/v3/client>
+This section documents many parts of the REST API provided by the Gravwell webserver. We also host an interactive [Swagger OpenAPI documentation instance](https://api.docs.gravwell.io).
```

Comment:
Do we really want to say "many parts" here? I would argue that this section actually documents a very limited portion of our rest API and _more_ details can be found with the Swagger documentation or the Golang client.

Reply:
It's a pretty good chunk of the API.   And we are on our way to the complete API.  I wouldn't worry about this kind of wording, otherwise we have to continually change it as we fill in the API.